server/routes/induction/common/previousWorkExperienceDetailController.ts

```diff
@@ -34,6 +34,14 @@ export default abstract class PreviousWorkExperienceDetailController extends Ind
       return next(createError(404, `Previous Work Experience type ${typeOfWorkExperience} not found on Induction`))
     }
 
+    // Check if we are in the midst of changing the main induction question set (e.g. from long route to short route)
```

Comment:
Not that I'm asking you to change this comment, but would it still be true if we were changing from short to long? I'm assuming we'd still need to do this?

Reply:
Indeed. As I'm sure you're aware, "e.g." literally means "for example" 😄 

The only difference would be the value within `updateInductionQuestionSet.hopingToWorkOnRelease`, but that's not relevant here.